pyproject.toml

```
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: Helmholtz Centre for Environmental Research (UFZ)
+# SPDX-FileCopyrightText: Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[project]
+name = "ansible-collection-toolkit"
+version = "5.3.0"
+description = "Add your description here"
```

```suggestion
description = "This collection provides production-ready Ansible roles used for providing services used in research and by research software engineers, but not exclusively."
```

(I know, since it's not a real Python package, the description doesn't matter probably. )